### DISCUSSION TITLE
Freeze Frame Option Have No Work When Transform Is Enabled

### DISCUSSION THREAD

**Discussion by JeromTWinL** [ppy/osu#21578] [Q&A] [answered]
Hi, I created a preset and added transform, freeze frame and other options, then i realised freeze frame is on and have no work for it while transform is on.

**Comment by mk56-spn** [comment, marked as answer]
Cannot repro on the latest release

TR+FR is giving me exactly what I'd expect to be seeing. Care to show a video of exactly what you are seeing 

 ![Screenshot_2022-12-08-13-08-46-075_sh.ppy.osulazer.jpg](https://user-images.githubusercontent.com/74463310/206443065-7fb6c82b-1362-4b22-a1a1-3fe6e3047090.jpg)

**Comment by JeromTWinL** [comment]
Just the approach circle.

**Reply by mk56-spn** [reply]
i'm confused and i don't think i can tell you much with this little information, its doing nothing other than what transform and Freeze frame already do separately but together, there is no functionality overlap. also please reply to the comment directly like this to avoid bloating the discussion

**Reply by mk56-spn** [reply]
if you mean you are only seeing the approach circle then that is due to using traceable

**Reply by JeromTWinL** [reply]
Yes, using only TR: circles appear from near the correct position

Using TR+FR: circles appear from the corner or sides.

**Reply by JeromTWinL** [reply]
But FR is mean to show all circles in one combination at the time.

(Sorry for my bad english)

**Reply by mk56-spn** [reply]
ah i understand what you mean now, thats just how FR works, it lengthens preempt time, and transform uses preempt to calculate distance, probably going to be another case of : https://github.com/ppy/osu/issues/21471 where mod order decides outcome. cant say much else might change in the future but this is just an unintended side effect of the mods sharing the information they work with
